Makefile

```diff
@@ -68,6 +68,7 @@ docker_test_integration:
 .PHONY: docker_test_lint
 docker_test_lint:
 	docker run --rm -it \
+	  -e ENABLE_BPMETADATA \
```

Comment:
Let's remove this as discussed so changes are not overwritten until tooling is ready

Reply:
I assume you mean to remove this from docker_generate_docs command which is responsible for generating metadata. I removed it from there. This is used for validation only and we can keep this here!

Reply:
Ah yes, got confused with the diff. You are right lets keep it.